src/commands/push.ts

```diff
@@ -87,7 +91,11 @@ export default class Push extends Command {
   }
 
   private async post_service_to_api(dto: CreateServiceVersionInput, account_id: string): Promise<any> {
-    const { data: service_digest } = await this.app.api.post(`/accounts/${account_id}/services`, dto);
-    return service_digest;
+    try {
+      const { data: service_digest } = await this.app.api.post(`/accounts/${account_id}/services`, dto);
+      return service_digest;
+    } catch (err) {
+      throw new Error(err.response.data.message);
```

Comment:
I'd prefer to remove all of these try/catch and let the base command catch method handle these types of errors. It might already.

Reply:
Lets update it there - then it'll look better for all api failures. Remove the getFormattedJSON method and replace it with something nice like:
```ts
const error_data = err.response.data;
this.error(chalk.red(`status: ${error_data.statusCode}\npath: ${error_data.path}\nmessage: ${error_data.message}`));
```